tox.ini

```diff
@@ -121,7 +121,7 @@ commands =
 
 [testenv:cover]
 basepython = python2.7
-passenv = NUMPROCESSES
+passenv = TRAVIS
```

Comment:
In addition to setting this here, I think we should set it in the `[testenv]` environment and inherit it in all of the `[testenv:py27-*oldest]` environments like we do for `setenv`. This should cause the variable to be passed through for all relevant Python 2.7 tests.